.fixtures.yml

```diff
@@ -20,7 +20,9 @@ fixtures:
     xinetd:        "https://github.com/puppetlabs/puppetlabs-xinetd"
     certs:         "https://github.com/theforeman/puppet-certs.git"
     qpid:          "https://github.com/theforeman/puppet-qpid.git"
-    pulpcore:      "https://github.com/theforeman/puppet-pulpcore.git"
+    pulpcore:
+      repo: "https://github.com/evgeni/puppet-pulpcore.git"
+      branch: "pulp-cli"
```

Comment:
this is just here to run tests before https://github.com/theforeman/puppet-pulpcore/pull/252 is merged ;)